src/dht/session.h

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2019 Duality Blockchain Solutions Developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef DYNAMIC_DHT_SESSION_H
+#define DYNAMIC_DHT_SESSION_H
+
+#include "dht/datarecord.h"
+#include "dht/sessionevents.h"
+#include "sync.h"
+
+#include "libtorrent/alert.hpp"
+#include "libtorrent/alert_types.hpp"
+#include "libtorrent/fwd.hpp"
+#include "libtorrent/session.hpp"
+#include "libtorrent/session_status.hpp"
+
+#include <map> // for std::map and std::multimap
+
+class CChainParams;
+class CConnman;
+class CEvent;
+class CKeyEd25519;
+class CLinkInfo;
+class CMutableData;
+class CMutableGetEvent;
+
+typedef std::pair<int64_t, CEvent> EventPair;
+typedef std::multimap<int, EventPair> EventTypeMap;
+typedef std::map<std::string, CMutableGetEvent> DHTGetEventMap;
+
+static constexpr int DHT_GET_ALERT_TYPE_CODE = 75;
```

Comment:
Might want to order these either alphabetically or by number

Reply:
Thanks, see https://github.com/duality-solutions/Dynamic/commit/a86dfc80a0837f2de3b60c9baf738b13abf25730